docs/_articles/en/einstein/einstein-setup.md

```diff
@@ -0,0 +1,98 @@
+---
+title: Einstein for Developers Setup
+lang: en
+---
+
+## Set Up Overview
+
+A Salesforce admin or user with the appropriate permissions can install Einstein for Developers in a supported Salesforce org. A developer with access to the org can then:
+
+1.  Install the extension in VS Code.
+2.  Connect to the org to use the extension.
+
+### Required Salesforce Editions and User Permissions
+
+**Available in**: Developer, Enterprise, Partner Developer, Performance and Unlimited Editions.
+
+**Inoperable in**: Group, Professional and Essentials Editions
+
+### User Permissions Needed
+
+**Configure Einstein for Developers(beta)**: `CustomizeApplication` and `ModifyAllData`
+
+## Enable Einstein for Developers
+
+An Admin can enable Einstein for Developers in a supported Salesforce org by following the given steps.
+
+1. From Setup, enter Einstein for Developers in the Quick Find box, then select Einstein for Developers. If you don’t see Einstein for Developers as an option under Setup in your Developer Edition org, it was likely created before Einstein for Developers was introduced and doesn’t have the correct permissions. To continue, create another Developer Edition org.
+2. Enable Einstein for Developers, and then review and accept the license agreement.
+
+Einstein for Developers(beta) is activated in the org and all the users in the org have access to the feature.
+
+### Add Users to the Org
+
+An admin can add additional users to the org as needed:
+
+1. From Setup, enter Users in the Quick Find box, then select **Users**.
+2. Click **New User** or **Add Multiple Users**.
+3. Select the appropriate license type and profile based on the user’s role.
+4. Select the **Generate passwords and notify user via email** checkbox.
+5. Click **Save**.
+   This procedure generates an email inviting the new users into the org.
+
+### Visual Studio Code Version
+
+VS Code releases a new version each month with new features and important bug fixes. You must be on VS Code Version 1.76 or higher to run the Einstein for Developers extension. You can manually check for updates from **Help** **> Check for Updates** on Linux and Windows or **Code > Check for Updates** on macOS.
+
+## Get Started with Einstein for Developers
+
+A developer with access to an org enabled for Einstein for Developers can install the Einstein for Developers extensions in VS Code, connect to org and start using the extension.
+
+## Install Einstein for Developers Extensions
```

Comment:
Can we also include how to install the Salesforce Extension Pack + Einstein for Developers Extensions from within VS Code and Code Builder?